packages/css/src/components/file-input/file-input.scss

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/text-rendering";
+
+@mixin reset-button {
+  border: 0;
+  border-radius: 0; // Reset rounded borders on iOS devices
+  box-sizing: border-box;
+}
+
+@mixin ams-file-input-selector-button-forced-color-mode {
```

Comment:
Why is this a mixin?

Reply:
Because I was told in the button to make it a mixin, just copying previous instructions is guess? I'll make it not a mixin.

Reply:
Maybe wait a second. What was the reason this should be a mixin in the button PR?

Reply:
To prevent duplication probably. I thought about reset where anything reset-ish should go in a reset.

Reply:
Or maybe because we wanted to make a util maybe?

Reply:
Partially implementing a util would be kind of weird, imo. I don't think that's the reason. 

The mixin is used multiple times in Button, it's only used once here. Try to think about the reason code is written the way it is, before copying it. That can easily lead to bugs.